pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "ms-office-macro-bleach"
+version = "0.1.0"
+description = "Tool to purge and remove all macro and dynamic content from an MS Office file"
+authors = ["Patterbear <benjamin.mcgregor2002@gmail.com>"]
```

Comment:
Maybe you want to keep your name, but remove email not to receive a lot of spam and dox you.